deploy/cloudwash_job.yaml

```diff
@@ -31,6 +31,14 @@ objects:
                   - "-d"
                   - "azure"
                   - "--all"
+                volumeMounts:
+                  - name: config-volume
+                    mountPath: /opt/app-root/src/cloudwash/settings.yaml
+                    subPath: settings.yaml
```

Comment:
suspect of this config with mountPath having the file, but I'm going to accept at this point to run in SRE cluster.